relay/pushservice/client_token_db.py

```diff
@@ -28,23 +29,39 @@ class ClientTokenAlreadyExistsException(ClientTokenDBException):
 
 class ClientTokenDB:
     def __init__(self, engine) -> None:
-        Session = sessionmaker(bind=engine)
+        self._make_session = sessionmaker(bind=engine)
         Base.metadata.create_all(engine)
-        self.session = Session()
+
+    @contextmanager
+    def session(self):
+        """Provide a transactional scope around a series of operations."""
+        session = self._make_session()
+        try:
+            yield session
+            session.commit()
```

Comment:
This always does a session commit, we use it later even if we only do a query, does that make sense?

Reply:
I actually copied that from the sqlalchemy documentation. We could also remove it and call commit inside the actual methods. I don't think there's a problem with calling commit even if we didn't write to the database.

From what I can tell this will also close the transaction that SQLAlchemy opened for us while querying the database.So, I think this something that we actually should do (doesn't matter in our case, because we close the session later on anyway).